pkg/cmd/cli/cmd/tag.go

```
@@ -303,7 +303,7 @@ func (o TagOptions) RunTag() error {
 		} else {
 			// The user wants to delete a spec tag.
 			if _, ok := target.Spec.Tags[destTag]; !ok {
-				glog.V(4).Infof("Destination tag %s/%s does not exist", o.destNamespace[i], destNameAndTag)
+				fmt.Printf("Destination tag %s/%s does not exist.\n", o.destNamespace[i], destNameAndTag)
```

We still have to return an error so I just return instead of logging.